cmd/egress/cmd.go

```diff
@@ -144,6 +144,11 @@ are set correctly before execution.
 
 				awsVerifier.Logger.Warn(context.TODO(), "Using region: %s", config.region)
 
+				if vei.Proxy.NoTls && vei.Proxy.Cacert != "" {
```

Comment:
I think it would make more sense for this to be done before line 94, so it applies to both AWS and GCP verifier blocks. For example, we load and parse the CA on line from the CLI flag, so instead we can just check `config.noTls` instead and skip the parsing of the CA altogether.

Reply:
Good suggestion. Done